jasmine/spec/GameSpec.js

```diff
@@ -0,0 +1,78 @@
+describe('Game', function() {
+
+  var game, frame, roll, frames, frameScore, runningTotal;
+
+  beforeEach(function() {
+    game = new Game();
+    roll = new Roll(game, frame);
+    frameScore = [];
+
+  });
+
+  describe('beginning of Game', function(){
+    it('starts a new game', function(){
+        expect(game).toEqual(game)
+    });
+    it('is empty before game starts', function(){
+        expect(game.frames).toEqual([]);
```

Comment:
This feels like testing state not behaviour. 

If we were to test various cases such as rolling a 7, rolling a 7 then a 3, rolling a 7 then a 3 and then a 5 (for example) then we can implicitly test this - because in order for the outcome of those tests to be correct then our start state must be as we expect it. 

Arguably, even if it isn't, this isn't something we need to test because that kind of state should be encapsulated - we should only be concerned with outcomes, not how those outcomes are achieved.